librustzcash/src/rustzcash.rs

```diff
@@ -1242,8 +1228,8 @@ pub extern "system" fn librustzcash_mmr_append(
     p_len: size_t,
     // New node pointer
     nn_ptr: *const [u8; zcash_mmr::MAX_NODE_DATA_SIZE],
-    // Return of root commitment (32 byte hash)
-    rt_ret: *mut u8,
+    // Return of root commitment
+    rt_ret: *mut [u8; 32],
     // Return buffer for appended leaves, should be pre-allocated of log2(t_len)+1 length
```

Comment:
(From review of #118 with @str4d.)

This is not well-defined when `t_len` is not a power of 2. Should it be `ceiling(log2(t_len)) + 1`, or is the `+1` intended to implement the ceiling?

Reply:
why binary log2 of `t_len` is not well-defined? +1 is for ceiling, yes
I does really need to specify upper bound only, so maybe mention it there

Reply:
I changed this to `ceiling(log2(t_len))` to be unambiguous.